nearlib/dev.js

```diff
@@ -31,7 +32,7 @@ module.exports = {
             options.accountId = aliceAccountName;
             options.key = aliceKey;
         }
-        const keyStore = new nearlib.InMemoryKeyStore();
+        const keyStore = options.deps && options.deps.keyStore ? options.deps.keyStore : new nearlib.BrowserLocalStorageKeystore();
```

Comment:
nit: `(options.deps && options.deps.keyStore) || new nearlib.BrowserLocalStorageKeystore();` is more idiomatic JS

Reply:
done